sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/uninstall/TLSCleanupStep.java

```diff
@@ -23,7 +23,18 @@
  */
 public class TLSCleanupStep extends AbstractStep {
 
-    private static final Pattern PATTERN = createSecretNamePattern();
+    private static final Pattern PATTERN;
+    static {
+        List<String> possibleSecretNames = Arrays.asList(
+                SecretNameGenerator.SECRET_NAME_CERTIFICATE,
```

Comment:
Shoudl the list of possible secret names not be the responsibility of the `SecretNameGenerator`? If it were an `enum`, for example, we could iterate over the values.